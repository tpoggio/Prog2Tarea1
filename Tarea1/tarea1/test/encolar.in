esVaciaCola
encolar 15 5.2
esVaciaCola
encolar 95 24.0
esVaciaCola
encolar 85 2.0
# puede haber repetidos
encolar 85 2.0
encolar 25 67.0
encolar 35 -12.0
encolar 75 2.5
encolar 45 2.1
encolar 55 -24.0
encolar 85 32.0
# esta llena
encolar 10 10.0
Fin
