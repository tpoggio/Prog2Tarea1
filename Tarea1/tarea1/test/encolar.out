1>La cola está vacia. 
2>Encolar. 
3>La cola no está vacia. 
4>Encolar. 
5>La cola no está vacia. 
6>Encolar. 
7>#  puede haber repetidos.
8>Encolar. 
9>Encolar. 
10>Encolar. 
11>Encolar. 
12>Encolar. 
13>Encolar. 
14>Encolar. 
15>#  esta llena.
16>Encolar. 
17>Fin.
